libmscore/page.cpp

```diff
@@ -36,38 +36,48 @@ namespace Ms {
-      PaperSize("A0",        MM(841),  MM(1189)),
-      PaperSize("A1",        MM(594),  MM(841)),
-      PaperSize("A2",        MM(420),  MM(594)),
-      PaperSize("A3",        MM(297),  MM(420)),
-      PaperSize("A5",        MM(148),  MM(210)),
-      PaperSize("A6",        MM(105),  MM(148)),
-      PaperSize("A7",        MM(74),   MM(105)),
-      PaperSize("A8",        MM(52),   MM(74)),
-      PaperSize("A9",        MM(37),   MM(52)),
-      PaperSize("A10",       MM(26),   MM(37)),
-      PaperSize("B0",        MM(1000), MM(1414)),
-      PaperSize("B1",        MM(707),  MM(1000)),
-      PaperSize("B2",        MM(500),  MM(707)),
-      PaperSize("B3",        MM(353),  MM(500)),
-      PaperSize("B4",        MM(250),  MM(353)),
-      PaperSize("B6",        MM(125),  MM(176)),
-      PaperSize("B7",        MM(88),   MM(125)),
-      PaperSize("B8",        MM(62),   MM(88)),
-      PaperSize("B9",        MM(44),   MM(62)),
-      PaperSize("B10",       MM(31),   MM(44)),
-      PaperSize("Comm10E",   MM(105),  MM(241)),
-      PaperSize("DLE",       MM(110),  MM(220)),
-      PaperSize("Folio",     MM(210),  MM(330)),
-      PaperSize("Ledger",    MM(432),  MM(279)),
-      PaperSize("Tabloid",   MM(279),  MM(432)),
-      PaperSize(0,           MM(1),    MM(1))   // mark end of list
+      PaperSize(QT_TRANSLATE_NOOP("paperSizes","Custom"),  MM(1),      MM(1)),
+      PaperSize("Letter (8.5 x 11 in)",                    8.5,        11),
+      PaperSize("Legal (8.5 x 14 in)",                     8.5,        14),
+      PaperSize("Tabloid (11 x 17 in)",                    11,         17),
+      PaperSize("A4 (210 x 297 mm)",                       MM(210),    MM(297)),
+      PaperSize("B5 (176 x 250 mm)",                       MM(176),    MM(250)),
+      PaperSize("Statement (5.5 x 8.5)",                   5.5,        8.25),
+      PaperSize("Executive (7.25 x 10.5 in)",              7.25,       10.5),
+      PaperSize(QT_TRANSLATE_NOOP("paperSizes","Concert Part (9 x 12 in)"),
+                9,          12),
+      PaperSize(QT_TRANSLATE_NOOP("paperSizes","Flip Folder (6.75 x 5.25 in)"),
+                6.75,       5.25),
+      PaperSize(QT_TRANSLATE_NOOP("paperSizes","Choral Octavo (6.75 x 10.5 in)"),
+                6.75,       10.5),
+      PaperSize(QT_TRANSLATE_NOOP("paperSizes","Hymn (5.75 x 8.25 in)"),
+                5.75,       8.25),
+      PaperSize("A0 (841 x 1189 mm)",                      MM(841),    MM(1189)),
+      PaperSize("A1 (594 x 841 mm)",                       MM(594),    MM(841)),
+      PaperSize("A2 (420 x 594 mm)",                       MM(420),    MM(594)),
+      PaperSize("A3 (297 x 420 mm)",                       MM(297),    MM(420)),
+      PaperSize("A5 (148 x 210 mm)",                       MM(148),    MM(210)),
+      PaperSize("A6 (105 x 148 mm)",                       MM(105),    MM(148)),
+      PaperSize("A7 (74 x 105 mm)"",                       MM(74),     MM(105)),
```

Comment:
You can see there is one " that shouldn't be there. The syntax coloring on github chokes on it